scripts/install_platform_driver_configs.py

```diff
@@ -39,11 +39,9 @@
 import os
 import glob
 
-from volttron.platform.agent.known_identities import CONFIGURATION_STORE, PLATFORM_DRIVER
-from volttron.platform import get_address
+from volttron.platform.agent.known_identities import CONFIGURATION_STORE, PLATFORM_DRIVER, PLATFORM
 from volttron.platform.keystore import KeyStore
-from volttron.platform.agent.utils import execute_command
-from volttrontesting.fixtures.volttron_platform_fixtures import build_wrapper
+from volttron.platform.vip.agent.utils import build_agent
 from argparse import ArgumentParser, RawTextHelpFormatter
```

Comment:
Please check import order as argparse should be before any of the volttron items.

Reply:
Will do.